examples/i3s/deck.gl/app.js

```diff
@@ -108,17 +130,36 @@ export default class App extends PureComponent {
     ];
   }
 
+  _renderStats() {
+    // TODO - too verbose, get more default styling from stats widget?
```

Comment:
Nit: Any chance to get this merged back as defaults into probe.gl, this verbose code is now in more and more examples...

Reply:
Follow up PR.